### ISSUE TITLE
just crashes

### ISSUE THREAD

**Issue by aep** [bart-kneepkens/OpenHeosControl#1]
````
Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: com/google/gson/GsonBuilder
	at Connection.TelnetConnection.connect(TelnetConnection.java:49)
	at SystemCommands.HeosSystem.<init>(HeosSystem.java:56)
	at gui.Main.connectButtonActionPerformed(Main.java:168)
	at gui.Main.access$400(Main.java:26)
	at gui.Main$5.actionPerformed(Main.java:104)
	at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
	at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
	at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
	at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
	at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
	at java.awt.Component.processMouseEvent(Component.java:6533)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.ClassNotFoundException: com.google.gson.GsonBuilder
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 41 more

````

**Comment by bart-kneepkens** [comment]
@aep can you please describe where this crash occurs to you? Are you trying to run the binary or do you have the project open in your IDE? If the latter, please make sure you've done a clean & build

**Comment by bart-kneepkens** [comment]
Can not replicate. Seems to be a linking issue. Make sure to do `mvn clean install` to make sure maven installs all dependencies that are declared in the `pom.xml` file.

**Comment by aep** [comment]
yeah, that was with the binary release https://github.com/bart-kneepkens/OpenHeosControl/releases/tag/v0.1

**Comment by bart-kneepkens** [comment]
I see. The .jar does indeed not work. I'll have to see if I'm creating the .jar the wrong way.

**Comment by bart-kneepkens** [comment]
This issue is caused by Maven not packing the dependencies by default.

It was fixed in [#fc776fe](https://github.com/bart-kneepkens/OpenHeosControl/commit/fc776fe891a594f0fd25e464b011a3d5b94b8f26).

Please see release 0.2 and check if this problem persists.
